File: programmers/난이도별/level01.약수의_합/JiaLee0707.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    
    for(int i=n; i>0; i--) {
        if(n%i==0) {
            answer += i;
        }
    }
    
    return answer;
}

File: programmers/난이도별/level01.최소직사각형/JiaLee0707.js
function solution(sizes) {
    var answer = 0;
    let max = [0,0];
    
    sizes = sizes.map((size) => size[1] > size[0] ? [size[1], size[0]] : size);
    
    for (const size of sizes) {
        if (size[0] > max[0]) max[0] = size[0];
        if (size[1] > max[1]) max[1] = size[1];
    }
    
    answer = max[0] * max[1];
    
    return answer;
}
